--- app/tests/talk/orga/views/test_orga_views_submission.py
+++ app/tests/talk/orga/views/test_orga_views_submission.py
@@ -303,9 +303,9 @@
     assert submission.speakers.count() == 1
 
     if user == "EMAIL":
-        data = {"email": other_orga_user.email}
+        data = {"email": other_orga_user.email, "biography": "bio"}
     else:
-        data = {"email": "some_unused@mail.org", "name": "New Speaker"}
+        data = {"email": "some_unused@mail.org", "name": "New Speaker", "biography": "bio"}
 
     response = orga_client.post(
         submission.orga_urls.speakers,
